e2e/pnpm_lockfiles/lockfile-test.bzl

```diff
@@ -12,11 +12,33 @@ def lockfile_test(name = "lockfile", node_modules = "node_modules"):
     build_test(
         name = name,
         targets = [
+            # The full node_modules target
             ":%s" % node_modules,
+
+            # Direct deps
             ":%s/@aspect-test/a" % node_modules,
             ":%s/@aspect-test/b" % node_modules,
             ":%s/@aspect-test/c" % node_modules,
-            ":%s/sharp" % node_modules,
+            ":%s/rollup" % node_modules,
             ":%s/uvu" % node_modules,
+
+            # Targets within the virtual store...
+            # Direct dep targets
+            ":.aspect_rules_js/node_modules/@aspect-test+a@5.0.2",
+            ":.aspect_rules_js/node_modules/@aspect-test+a@5.0.2/dir",
+            ":.aspect_rules_js/node_modules/@aspect-test+a@5.0.2/pkg",
+            ":.aspect_rules_js/node_modules/@aspect-test+a@5.0.2/ref",
+
+            # Direct deps with lifecycles
+            ":.aspect_rules_js/node_modules/@aspect-test+c@2.0.2/lc",
+            ":.aspect_rules_js/node_modules/@aspect-test+c@2.0.2/pkg_lc",
+
+            # TODO: differs across lockfile versions
+            # Direct deps from custom registry
+            # ":.aspect_rules_js/node_modules/@types+node@registry.npmjs.org+@types+node@16.18.11",
```

Comment:
These ones are interesting where we generate different targets because the lockfiles are formatted different. Do we care?